src/preamble/01_what_do_we_mean_when_we_say_cabal.md

```diff
@@ -1 +1,73 @@
 # What do we mean when we say cabal?
+
+Cabal is an umbrella term that can refer to either cabal-the-spec (.cabal
+files), cabal-the-library (code that understands .cabal files), or
+cabal-the-tool (the cabal-install package which provides the cabal executable).
+
+## tl;dr
+
+1. [cabal-install](#cabalInstall) is the command line utility to help configure,
+   compile and install Haskell libraries and programs.
+
+2. [The Cabal build system](#cabailBuildSystem), a specification for defining
+   Haskell packages, together with a library for performing builds. This build
+   system is used by cabal-install and other package managers (stack for eg).
+   For eg: parsing .cabal files (among others).
```

Comment:
I was thinking I'd just add one or two-line summaries here. I'll add some more details here then!

Reply:
@srirajshukla I am fine with this tl;dr section being small. But I think if you introduce a concept (like parsing a .cabal file) you need to fully flesh it out, otherwise just don't mention it.